packages/local_contexts/lc-cvoc-conf.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "field-name": "LCProjectUrl",
+    "term-uri-field": "LCProjectUrl",
+    "js-url": "https://gdcc.github.io/dataverse-external-vocab-support/packages/local_contexts/local_contexts.js",
+    "protocol": "localcontexts",
+    "retrieval-uri": "https://localcontextshub.org/api/v1/projects/{0}",
+    "allow-free-text": true,
```

Comment:
Not sure this is desired or that it is supported by the script right now - should it be false?